gcp-jobs/entity-bn/devops/gcp/clouddeploy.yaml

```diff
@@ -30,7 +30,6 @@ serialPipeline:
       deploy-project-id: "a083gt-dev"
       job-name: "business-entity-bn-dev"
       cloudsql-instances: "a083gt-dev:northamerica-northeast1:businesses-db-dev"
-      run-command: "./run.sh"
```

Comment:
Are you sure you don't need the run command in the job?

Reply:
Okay I've put these back in and fixed the issue with the run.sh files